docs/docs/general/06-Tutorials/google-cat-search.md

```diff
@@ -118,7 +118,7 @@ describe('jest with askui', () => {
       .typeIn("cat")
       .textfield()
       .contains()
-      .icon().withText('microphone')
+      .icon('microphone')
```

Comment:
Does that already work? Was not aware of it or forgot about it.

Reply:
So I interpret the resolving of the thread as "it works"

Reply:
A mistake from my side on the resolve. Once the issue with relational selectors is fixed I will merge.

Reply:
With all the different PRs this got mixed up with the `matching()` one. Really sorry for the confusion!